bindings/rust/evmc-vm/src/lib.rs

```diff
@@ -308,4 +480,75 @@ mod tests {
             }
         }
     }
+
+    unsafe extern "C" fn get_dummy_txctx(context: *mut ffi::evmc_context) -> ffi::evmc_tx_context {
```

Comment:
Any reason for the shorthand `txctx`? :)

Reply:
its a test helper leave me alone :(
will fix it in a bit

Reply:
Can we name this nicely?